src/Blocks/BlockCocoaPod.h

```diff
@@ -24,12 +24,21 @@ class cBlockCocoaPodHandler :
 
 	virtual bool CanBeAt(cChunkInterface & a_ChunkInterface, const Vector3i a_RelPos, const cChunk & a_Chunk) override
 	{
+		if (!cChunkDef::IsValidHeight(a_RelPos.y - 1))
```

Comment:
Same thing, we aren't reading a block at `a_RelPos.y - 1`.

Reply:
Yep, the -1 slipped through with the previous re-amendments